x/zkauth/keeper/keeper.go

```diff
@@ -118,3 +121,8 @@ func (k Keeper) GetJWK(endpoint string) (*http.Response, error) {
 
 	return resp, nil
 }
+
+func (k Keeper) CreateJWKFileName(name types.OidcProvider) string {
+	fileNamePattern := strings.Replace(JwkFileName, ".", "-%s.", 1)
+	return fmt.Sprintf(fileNamePattern, name)
```

Comment:
It seems like it could be written more simply.
```suggestion
        return fmt.Sprintf("jwk-%s.json", name)
```